vendor/github.com/google/go-cmp/cmp/compare.go

```diff
@@ -36,7 +36,6 @@ import (
 	"strings"
```

Comment:
On a meta note, should we get rid of the vendor directory? Most other hashicorp repos have.